apps/client/src/utils/validation-message.ts

```diff
@@ -0,0 +1,30 @@
+export const displayNamePatten = /^[a-zA-Z0-9-_]+$/;
```

Comment:
if these regex not being used anymore, we can remove it

Reply:
use this pattern in the setting form validation